samwise.js

```diff
@@ -3,30 +3,28 @@
  *
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.